arfc-pres.tex

```diff
@@ -61,9 +61,11 @@
   \vskip0pt%
 }
 \makeatletter
+\setbeamertemplate{caption}[numbered]
```

Comment:
Maybe it would be helpful to add a comment here saying what this is doing/overriding.

Reply:
I think that's a good idea!

Reply:
I actually think the line speaks for itself (it changes the beamer template to numbered captions.) But, a comment doesn't hurt unless it gets out of date with the code. . .

Reply:
That's true, it is more in line with the style of the preamble to have a comment, but it is a pretty explanatory snippet.

Reply:
I'm fine either way.